addons/website_livechat/static/src/components/discuss/discuss.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-inherit="mail.Discuss.content" t-inherit-mode="extension">
+        <xpath expr="//*[hasclass('o_Discuss_thread')]" position="before">
+            <t t-if="discuss.threadView.thread.visitor">
+                <VisitorBanner
+                    visitorLocalId="discuss.threadView.thread.visitor.localId"
```

Comment:
Don't need `threadView` indirection, source of truth for the thread view is actually discuss itself ```suggestion
                    visitorLocalId="discuss.thread.visitor.localId"
```